src/many/src/types/identity/cose.rs

```diff
@@ -226,7 +223,10 @@ impl Signer<CoseKeyIdentitySignature> for CoseKeyIdentity {
                 Algorithm::None => Err(Error::new()),
                 Algorithm::ECDSA => {
                     if self.hsm {
-                        let hsm = HSM_INSTANCE.lock().unwrap();
+                        let hsm = HSM::get_instance().map_err(|e| {
+                            eprintln!("HSM mutex poisoned {}", e);
```

Comment:
Please use `error!()` from the tracing module instead of `eprintln!`, here and everywhere else.

Reply:
Done. I took the liberty of removing every `eprintln!` from `many-rs`.